### PR TITLE
Fixed .glide error

### PR DESCRIPTION
**Pull request by pbielak**
Fixes error while building vagrant (`vagrant up`), when no glide repository is available in the host OS.

Summary of changes:
- ~~Instead of always syncing the .glide directory from the host and guest OS, it's done now only when this directory exists in the host OS~~
- On default, there is no syncing of the `.glide` directory between the host and guest OS
- For enabling syncing an environment variable must be set `SHARE_GLIDE_CACHE`

Testing done:
- The `vagrant up` command runs now fine.

### CODE REVIEW

**Comment by pbielak** [comment]
run integration tests

File: vagrant/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 First - install: [Vagrant](https://www.vagrantup.com/docs/installation/), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Then you just need to execute the following commands (they should work on any flavour of Linux):
 
 To run Vagrant development environment, the `$SWAN_DEVELOPMENT_ENVIRONMENT` variable must be set.
-
+If you want to use your local Glide repository, the `$SHARE_GLIDE_CACHE` variable must be set.
```
Comment (mpatelcz) [inline vagrant/README.md]: I would rephrase it to:
If you want to use you local Glide cache in the Vagrant development environment the `$SHARE_GLIDE_CACHE` variable must be set.

Comment (pbielak) [inline vagrant/README.md]: Done

**Comment by pbielak** [comment]
run integration tests

**Comment by pbielak** [comment]
run integration tests
